README.md

```diff
@@ -25,25 +25,36 @@ estimated backfill time @ 50rps = 24-36 hours on M1 Macbook (~10x speedup)
   - (possible) continued backwards compatibility with subgraph
   - support indexing subset of data, i.e. only domains under parent node
 
-### todo
+#### known bugs
+
+- [ ] there's an account `0x` in the database, which is definitely a bug — perhaps all addresses should go through `viem/getAddress` before being handled (presumably ponder did this inthe background...)
+- [ ] root domain is not `is_migrated`, probably should be
```

Comment:
Confirmed. Here's an easy UI for building a query against the ENS Subgraph: https://thegraph.com/explorer/subgraphs/5XqPmWe6gjyrJtFn9cLy237i4cWw2j9HcUJEXsP5qGtH?view=Query&chain=arbitrum-one#query-subgraph

Here's a query I ran. The id here is the id of the ENS Root.

```
{
  domains(
    where: {id: "0x0000000000000000000000000000000000000000000000000000000000000000"}
  ) {
    id
    isMigrated
  }
}
```

The above query returns:

```
{
  "data": {
    "domains": [
      {
        "id": "0x0000000000000000000000000000000000000000000000000000000000000000",
        "isMigrated": true
      }
    ]
  }
}
```